modules/xfem/src/efa/EFAElement2D.C

```diff
@@ -1055,6 +1055,19 @@ EFAElement2D::createChild(const std::set<EFAElement *> & CrackTipElements,
         EFAPoint origin_to_point = p - origin;
         EFAPoint origin2_to_point = p - origin2;
 
+        // shift the origin
+        if (std::abs(origin_to_point * normal) < Xfem::tol)
+        {
+          EFAPoint shift(Xfem::tol * 10.001, Xfem::tol * 9.999, 0.0);
```

Comment:
Look like this constants are repeated in several places, can you declare them just once and use them?